core/ra2.js

```diff
@@ -0,0 +1,259 @@
+import "../vendor/aes.js";
+import "../vendor/sha1.js";
+import "../vendor/jsencrypt.js";
```

Comment:
We no longer support Internet Explorer, so please have a look at using SubtleCrypto instead so we can avoid these extra scripts.